boshrelease/pipeline.yml

```diff
@@ -205,11 +208,41 @@ jobs:
 
   - name: rc
     public: true
+    serial: true
     plan:
     - do:
       - aggregate:
           - { get: git,     trigger: true,  passed: [testflight] }
           - { get: version, trigger: true, params: {pre: rc} }
+      - task: release-notes
```

Comment:
Does this show up in the GUI; or is it collapsed and a user won't necessarily see the message?

Reply:
Not sure what to do about it if its the latter.

Reply:
It's the latter, but it's the best solution we've got available to us.